make/modules/java.base/lib/CoreLibraries.gmk

```diff
@@ -162,6 +162,14 @@ ifneq ($(USE_EXTERNAL_LIBZ), true)
           zcrc32.c \
           zutil.c \
       )
+
+  # Do not include these libz objects in the static libjli library.
+  # When statically linking the java launcher with all JDK and VM
+  # static libraries, we use the --whole-archive linker option.
+  # The duplicate objects in different static libraries cause linking
+  # errors due to duplicate symbols.
+  LIBJLI_STATIC_EXCLUDE_OBJS := \
+      inflate.o inftrees.o inffast.o zadler32.o zcrc32.o zutil.o
```

Comment:
This is the same list as the LIBJLI_EXTRA_FILES above. Would be good to avoid the duplication.

Reply:
> This is the same list as the LIBJLI_EXTRA_FILES above. Would be good to avoid the duplication.

@erikj79 Addressed in the extracted https://github.com/openjdk/jdk/pull/13768, which contains .a related changes only (without linking 'javastatic' part), thanks.